common-java-root/build.gradle

```diff
@@ -0,0 +1,52 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+project.ext.vstsUsername = System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME") != null ? System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME") : project.findProperty("vstsUsername")
+project.ext.vstsPassword = System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_ACCESSTOKEN") != null ? System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_ACCESSTOKEN") : project.findProperty("vstsMavenAccessToken")
+
+buildscript {
+    apply from: rootProject.file("../gradle/versions.gradle")
+}
+
+allprojects {
+    /*
+    Order of repositories is important.
+     */
+    repositories {
+        /*
+        This is the project main repository and contains all of the packages we publish.
+        This azure devops feed also has an upstream to Maven Central as well as several other
+        internal azure devops respositories
+
+        NOTE: This repository is not currently available to our public CI (Travis)
+         */
+        maven {
+            name "vsts-maven-adal-android"
+            url 'https://identitydivision.pkgs.visualstudio.com/IDDP/_packaging/Android/maven/v1'
+            credentials {
+                username project.vstsUsername
+                password project.vstsPassword
+            }
+        }
+        //Required for google published packages not published to maven central
+        google()
+        /*
+        Required for Android Studio / Android Tools (for example: linter) packages not published
+        to maven central.
+
+        NOTE: When building in Travis packages will be pulled from here
+         */
+        mavenCentral()
+        /*
+        Required since the DUO sdk is not published to any of the other large android
+        repositories... which is odd.
+
+        NOTE: This is required for Travis.
```

Comment:
We're not using travis anymore... is this still true?

Reply:
Removed: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1468/commits/bc39b8c49e006abc84bdb1f8683f639c9265bc51